src/main/scala/Main.scala

```diff
@@ -1,8 +1,84 @@
+
+
+import newmodel.Package
+import newmodel._
+
+import scala.collection.mutable.ListBuffer
+
 /**
  * Created by yaroslav on 11.05.2015.
  */
 object Main {
-  def main(args:Array[String]) ={
-    println("hello world")
+
+  object DummyIndex extends Index{
+    val docElements = ListBuffer.empty[DocElement]
+    override def getConversionsFor(id: ID): Seq[ImplicitConversions] = ???
+
+    override def getKnownSubclasses(id: ID): Seq[ID] = ???
+
+    override def getShadowedConversionsFor(id: ID): Seq[ImplicitConversions] = ???
+
+    override def getLinearSupertypesBy(id: ID): Seq[ID] = ???
+
+    override def getCompanionFor(id: ID): ID = ???
+
+    override def getDocElementBy(id: ID): DocElement = docElements.find((x) => x.id == id).get
```

Comment:
@clhodapp yeah I was initially thinking about the data representation being opinionated as well, but then again, having any representation and having opinionated methods on top of it to fetch the relevant data should actually be enough. And that's what scaladoc already does doesn't it? Only the Scaladoc is tied to the Scala ASTs and (maybe?) to html. By making the scaladoc model, or a set of additional methods on top of it more opinionated about output, but less opinionated about the format, one could probably get what we need? There maybe a purely organizational usefulness in having a separate, stripped down AST-like structure (which could be opinionated towards output), but it would mean replicating parts of the AST. Meta seems like it is already doing a longer term effort to provide a new, cleaned up AST for Scala. How do we see this in relation to meta, in case we pursue the docs AST?

/cc @MasseGuillaume